### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by emy**

### CODE REVIEW

File: commands/General/Image/sauce.js [hunk 1]
```diff
@@ -18,17 +18,16 @@ module.exports = class extends Command {
...
+    const expr = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&/=]*)/;
     if (!img.match(expr)) return msg.sendError(lang.get('NO_VALID_URL'));
     const data = await snClient(img);
     if (!data) return msg.sendError('NO_SOURCE');
     const display = new RichDisplay();
     const noInfo = lang.get('NO_INFORMATION');
     const animeSources = ['AniDB'];
-    const booruSources = ['Danbooru'];
-    
+    // const booruSources = ['Danbooru'];
```
Comment (Zelle97) [inline commands/General/Image/sauce.js:28]: If you dont need it anymore delete it. No need to comment out unused code.

File: commands/General/Voice/play.js [hunk 2]
```diff
@@ -110,7 +109,7 @@ module.exports = class extends Command {
       return msg.sendError('NOT_SAME_CHANNEL');
     }
     while (tracks.length > 0) {
-      // console.log('added song:', tracks[0].info.title);
```
Comment (Zelle97) [inline commands/General/Voice/play.js:113]: Same here with unused code thats commented out.

**Review by Zelle97 — CHANGES_REQUESTED** [review]
Except for the unused code thats in the comments approved.

File: commands/General/Image/sauce.js (hunk 1 above)
Comment (emy) [reply commands/General/Image/sauce.js:28]: i don't need it *yet*

File: commands/General/Voice/play.js (hunk 2 above)
Comment (emy) [reply commands/General/Voice/play.js:113]: this is a comment about how to access the track data in the future 🤔

**Review by Zelle97 — APPROVED** [review]
Comments are there for documentary reasons. Approved for this pr.

**Pull request merged**